schedule/yam/test_cases/minimal+role_minimal.yaml

```diff
@@ -4,10 +4,6 @@ description: >
   SLE 15 specific registered installation with minimal role selected.
   Installation is validated by successful boot and that YaST does not report
   any issue.
-vars:
```

Comment:
Please check and add these removed vars in yaml file to corresponding test suites.

Reply:
Already done, added them in MR, and added the related MR link to the description.